using-valtimo/document/custom-case-columns.md

```diff
@@ -0,0 +1,162 @@
+
+## Default columns
+
+In the environment file, the following default inline code is found for the case overview columns:
+
+#### **`environment.ts`**
+  ```typescript
+...
+
+const defaultDefinitionColumns: Array<DefinitionColumn> = [
+    {
+        propertyName: 'sequence',
+        translationKey: 'referenceNumber',
+        sortable: true,
+    },
+    {
+        propertyName: 'createdBy',
+        translationKey: 'createdBy',
+        sortable: true,
+    },
+    {
+        propertyName: 'createdOn',
+        translationKey: 'createdOn',
+        sortable: true,
+        viewType: 'date',
+        default: true,
+    },
+    {
+        propertyName: 'modifiedOn',
+        translationKey: 'lastModified',
+        sortable: true,
+        viewType: 'date',
+    },
+];
+
+...
+
+export const environment: ValtimoConfig = {
+    ...
+    defaultDefinitionTable: defaultDefinitionColumns,
+    ...
+};
+  ```
+
+## Custom columns
+
+This above code can be used as a basis for custom columns. First add the line `customDefinitionTables: {}` below the
+line `defaultDefinitionTable: defaultDefinitionColumns`:
+
+#### **`environment.ts`**
+  ```typescript
+...
+
+export const environment: ValtimoConfig = {
+    ...
+    defaultDefinitionTable: defaultDefinitionColumns,
+    customDefinitionTables: {}
+    ...
+};
+  ```
+
+Inside the brackets custom columns can then be defined:
+
+#### **`environment.ts`**
+  ```typescript
+...
+
+export const environment: ValtimoConfig = {
+    ...
+    defaultDefinitionTable: defaultDefinitionColumns,
+    customDefinitionTables: {
+        'document-definition-name' : [
+            {
+                propertyName: '$.firstName',
+                translationKey: 'document-definition-name.firstName',
+                sortable: false,
+                viewType: 'string',
+                default: true
+            },
+            {
+                propertyName: 'modifiedOn',
+                translationKey: 'lastModified',
+                sortable: true,
+                viewType: 'date'
+            }
+        ]
+    }
+    ...
+};
+  ```
+
+For each case for which custom columns are to be defined, an array is added with the desired columns. The document
+definition id is used as a key to point to this array, i.e.:  `'document-definition-name': []`.
+
+In this array, for each column an object is added with the following properties:
+
+- `propertyName`: The key from the document definition that is to be used to display column data. Default properties
+like `sequence` can be written plainly. Properties nested inside like `firstName` use a `$.` notation, i.e.: 
+`$.firstName`.
+- `translationKey`: The path to a translation in the translation files. In the above example, a combination of the
+document definition name and the key from the document definition is used.
+- `sortable`: A `boolean` value, which states whether the column should be sortable. Only enable this if the back-end
+supports sorting on this property.
```

Comment:
Maybe we should list what the back-end can sort on. as far as i can see it should be the following:
- createdOn
- modifiedOn
- createdBy (maybe we shouldn't include this because it would sort by the id, not the name of the user)
- sequence
- All JSON path expressions on the content